airflow/models/dag.py

```diff
@@ -858,7 +868,7 @@ def infer_automated_data_interval(self, logical_date: datetime) -> DataInterval:
         DO NOT use this method is there is a known data interval.
         """
         timetable_type = type(self.timetable)
-        if issubclass(timetable_type, (NullTimetable, OnceTimetable)):
+        if issubclass(timetable_type, (NullTimetable, OnceTimetable, ContinuousTimetable)):
```

Comment:
This is not needed. The function is designed for older runs created prior to Airflow 2.2 (when timetable was introduced), and it should not be possible for a `ContinuousTimetable` to reach here.